common-rest/src/main/java/org/uniprot/api/rest/download/queue/QueueConsumerConfig.java

```diff
@@ -0,0 +1,38 @@
+package org.uniprot.api.rest.download.queue;
+
+import org.springframework.amqp.core.MessageListener;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.listener.MessageListenerContainer;
+import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * @author sahmad
+ * @created 22/11/2022
+ */
+@Configuration
+@Profile({"asyncDownload"})
+public class QueueConsumerConfig {
+
+    @Bean
+    public MessageListenerContainer messageListenerContainer(
+            ConnectionFactory connectionFactory,
+            @Qualifier("Consumer") MessageListener messageListener,
```

Comment:
I don't like the name 'Consumer'. It doesn't make it specific (If some other comes, that would also be a consumer). 
What if something like UniProtKBMessageListener?

Reply:
We need to change the Qualifier